### PR TITLE
Further readme improvements

### PR DESCRIPTION
**Pull request by LEARAX**
The readme still has a lot of small errors and inconsistencies. With it being the first thing newcomers to Lutris might see, I figured it could use a bit of revising.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -2,21 +2,17 @@
...
+Lutris is an open source gaming platform that makes gaming on Linux easier by
+managing, installing and providing optimal settings for games.
 
-Lutris does not sell games; you have to provide your own copy of the games
-unless they are open source or freeware.
-Games can be installed anywhere on your system; Lutris does not impose
-anything.
+Lutris does not sell games, and you must provide your own copy of a game
+unless it is open source/freeware.
+Instead, Lutris utilizes scripts called "runners" to provide a large library of
```
Comment (strycore) [inline README.rst]: nope, scripts are just scripts, runners are the programs that run the games (wine, retroarch, mame, ...) and the code associated with them.

File: README.rst [hunk 2]
```diff
@@ -56,62 +52,57 @@ We currently support the following runners:
...
-perform a series of actions on them.
-The syntax of installers is described in ``docs/installers.rst``, and is also
-available on `lutris.net <https://lutris.net>`_ when writing installers.
+Lutris installations are fully automated through runners, which can be written
+in either JSON or YAML.
+The runner syntax is described in ``docs/installers.rst``, and is also
+available online at `lutris.net <https://lutris.net>`_.
 
-A web UI is planned to ease the creation of these scripts.
+A web UI is planned to ease the creation of runners.
```
Comment (strycore) [inline README.rst]: I was like… wth, I never said that!… but then I realized you changed scripts to runners :laughing:

File: README.rst [hunk 3]
```diff
@@ -136,24 +127,24 @@ identifier on the command line such as::
...
-  private storage, allowing you to reinstall them on all your devices)
-* Game saves sync
-* Community features (friends list, chat, multiplayer game scheduling)
+* GOG and Humble Bundle integration
+* TOSEC database integration
+* Management of personal game data, i.e. syncing games across devices using private cloud storage
+* Save syncing
+* Community features (friends list, chat, multiplayer game scheduling, etc.)
 * Controller configuration GUI (with xboxdrv support)
+* Web UI for editing runners
```
Comment (strycore) [inline README.rst]: no! :rofl:

**Comment by strycore** [comment]
Gonna merge this and just fix the runner confusion myself because those updates are just too valuable to be sitting in a PR

**Comment by LEARAX** [comment]
Ah, sorry. Completely forgot about this.

**Pull request merged**